using-valtimo/plugin/zaken-api/configure-zaken-api-plugin.md

```diff
@@ -45,3 +45,11 @@ When creating a process link the following properties have to be entered:
 
 An example process link configuration:
 ![Generate document process link](img/link-document-to-zaak.png)
+
+### Link uploaded document to zaak
+
+The **Link uploaded document to zaak** action takes a user uploaded Document stored in a Documenten API and links it to
```

Comment:
I don't think document should be capitalized, or are the words `user uploaded` particularly important, as this says something about what happened before this action (and therefore is irrelevant).